src/lib/client.cc

```diff
@@ -128,16 +128,48 @@ Client::GetBucket(const QString& bucketName, const QString& prefix,
 void
 Client::CreateBucket(const QString& name)
 {
-	ds3_request* request = ds3_init_put_bucket(name.toUtf8().constData());
-	LOG_INFO("PUT          BUCKET    "+m_endpoint+"/"+name);
-	ds3_error* ds3Error = ds3_put_bucket(m_client, request);
-	ds3_free_request(request);
+    ds3_request* request = ds3_init_put_bucket(name.toUtf8().constData());
```

Comment:
The browser loosely follows FreeBSD's style(9) guide and thus uses tabs.  All the indentation is based on 8 space tabs.  I personally don't care what coding style the browser uses as long as its consistent across the entire project.  Ryan's certainly free to decide to change it to something else, however, that should be done as part of a separate pull request.  For this pull request, all indentation should use tab characters to keep it consistent.